cmd/run.go

```diff
@@ -127,6 +158,12 @@ func parseCustomCommandsScript(script string) (parsedCommands [][]string) {
 		os.Exit(2)
 	}
 
+	if (!isRunningGlobal && foundGlobal) {
+		messageStyle := "\033[43m\033[30m"
+		defaultStyle := "\033[0m"
```

Comment:
This might be worth it's own issue/PR but I will start the discussion here -

would be great to have better output overall. Using something like this for colors: https://github.com/fatih/color

Or something like this for going crazy: https://github.com/buger/goterm (going crazy brings concerns on non TTY environments of course, so we should be careful going dow this road).